src/apps/chifra/pkg/cache/types.go

```diff
@@ -0,0 +1,25 @@
+package cache
+
+type unsignedLong = uint64
+type cString struct {
+	size    unsignedLong
+	content []byte
+}
+
+type cacheHeader struct {
+	deleted   uint64
+	schema    uint64
+	showing   uint64
+	className cString
+}
+
+func reverseBytes(original []byte) (reversed []byte) {
```

Comment:
Can you explain this? Is it needed? What for?

Reply:
Sure, I was surprised that we need it, too. So, when Go sees `[]byte`, it ignores byte order when using `binary.Read` and `binary.Write`. It treats `[]byte` as raw bytes. And because our files are little endian and Go is big endian, I have to reverse this slice

Reply:
There are GoLang routines to specify the endian (for example, here: https://github.com/TrueBlocks/trueblocks-core/blob/master/src/apps/chifra/pkg/index/chunk_data_appearance_record.go#L48.)

Could we not have used that?

I'd prefer to be very specific so there's no ambiguity.

Is that possible?